src/main/app/src/Components/Landscape/Modals/Item/Item.tsx

```diff
@@ -239,7 +247,21 @@ const Item: React.FC<Props> = ({ fullyQualifiedItemIdentifier, small, sticky })
               <ItemAvatar item={other} statusColor={status?.status || ''} />
             </IconButton>
           </ListItemIcon>
-          <ListItemText primary={primary} secondary={secondary} />
+          <ListItemText
```

Comment:
See above concerning color and frontend mapping.

Reply:
Frontend Mapping is resolved in new branch